### PR TITLE
Rounding

### LINKED ISSUE #729: The quantity in the UI has to be rounded to 100 precision
You cannot go more precise than 100-steps on BitMEX. To allow a proper hedging calculation we should apply the same on the taker UI.

### PR DESCRIPTION
**Pull request by rishflab**
closes #729

### CODE REVIEW

File: taker-frontend/src/components/Trade.tsx [hunk 1]
```diff
@@ -289,6 +297,7 @@ function Quantity({ min, max, onChange, quantity }: QuantityProps) {
                     min={min}
                     max={max}
                     default={min}
+                    step={100}
```
Comment (bonomat) [inline taker-frontend/src/components/Trade.tsx]: You could use the same variable here as well: 
```suggestion
                    step={quantityIncrement}
```

File: taker-frontend/src/components/Trade.tsx [hunk 2]
```diff
@@ -345,3 +354,11 @@ function Margin({ margin }: MarginProps) {
         </VStack>
     );
 }
+
+export function isEvenlyDivisible(numerator: number, divisor: number): boolean {
+    if (numerator % divisor == 0.0) {
```
Comment (bonomat) [inline taker-frontend/src/components/Trade.tsx]: Nit: 

```suggestion
    return (numerator % divisor === 0.0);
```

**Review by bonomat — APPROVED** [review]
👍

**Review by thomaseizinger — APPROVED** [review]
Nice work, thanks for adding tests!

**Pull request merged**